<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="uiWellTableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>1251</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="gridStyle">
    <enum>Qt::DotLine</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="uiGoButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>40</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="uiTimeLabel">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>6</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>61</y>
     <width>751</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="uiSample1Button">
      <property name="text">
       <string>Sample 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uiSample2Button">
      <property name="text">
       <string>Sample 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uiSample3Button">
      <property name="text">
       <string>Sample 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uiSample4Button">
      <property name="text">
       <string>Sample 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uiSample5Button">
      <property name="text">
       <string>Sample 5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uiSample1Button_6">
      <property name="text">
       <string>Blank</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
